engine/runtime/src/test/java/org/enso/interpreter/test/SignatureTest.java

```diff
@@ -872,7 +872,7 @@ public void returnTypeCheckOptInError() throws Exception {
       var res = plusChecked.execute("a", "b");
       fail("Expecting an exception, not: " + res);
     } catch (PolyglotException e) {
-      assertContains("expected `expression` to be Integer, but got Text", e.getMessage());
+      assertContains("expected `plusChecked` result to be Integer, but got Text", e.getMessage());
```

Comment:
What about the following error message:
```
expected the result of `plusChecked` method to be Integer, but got Text
```
Isn't that even better?

Reply:
sure it sounds good to me, I will amend when I will get around to moving this PR forward, thanks for the suggestion

Reply:
I just think that maybe `function` may be a more universal name here?

`method` fits well for `Type.foo` methods. But internal helper functions e.g.:
```
foo x =
    go i = ...
    go 0
```
are not really _method_s, are they?

So maybe 
```
expected the result of `plusChecked` function to be Integer, but got Text
```
?

Reply:
I don't know. I thought that everything *function-like* in Enso is just called *method*.

Reply:
I did
```
expected the result of `plusChecked` to be Integer, but got Text
```

avoiding the method/function controversy.

IMO it is readable.